tests/connect/standalone-x509-auth-002.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+Connect to MongoDB with SSL and X509 auth (stream context)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("STANDALONE_X509"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$SSL_DIR = realpath(__DIR__ . '/../../scripts/ssl/');
+
+$driverOptions = [
+    'context' => stream_context_create([
+        'ssl' => [
+            // libmongoc does not allow the hostname to be overridden as "server"
+            'allow_invalid_hostname' => true,
+            'allow_self_signed' => false, // "weak_cert_validation" alias
+            'cafile' => $SSL_DIR . '/ca.pem', // "ca_file" alias
+            'local_cert' => $SSL_DIR . '/client.pem', // "pem_file" alias
+            // TODO: this doesn't appear to have any effect. Does the PEM file not have a password?
+            'passphrase' => 'qwerty', // "pem_pwd" alias
```

Comment:
Forgot to remove this before merging. Will be addressed in https://github.com/mongodb/mongo-php-driver/pull/376.